src/components/SnackStack.tsx

```diff
@@ -0,0 +1,33 @@
+import { Stack, Snackbar } from '@mui/material';
+import { useSnackStack } from './SnackStackProvider';
+import SnackbarToast from './SnackToast';
+
+function SnackStack() {
+  const { toastsPack } = useSnackStack();
+
+  const firstToast = toastsPack[0];
+
+  return (
+    <Snackbar
+      open={!!firstToast}
+      autoHideDuration={null}
```

Comment:
`null` seems to be the default. Are you setting it on purpose? else I'd just remove